test/merge/test_rtrans_build.c

```diff
@@ -26,6 +26,11 @@ DEALINGS IN THE SOFTWARE.  */
 
 #include "../../bam_sort.c"
 
+// For tests, just return a constant that can be embedded in expected output.
+const char *samtools_version(void) {
```

Comment:
Better to adjust the _Makefile_ instead: add _test/test.o_ to the link recipes for _test/merge/test_rtrans_build_ and _test/merge/test_trans_tbl_init_ — similarly to the existing _test/merge/test_bam_translate_ recipe.

Reply:
Thanks. Done.